src/main/java/com/alipay/oceanbase/rpc/util/MonitorUtil.java

```diff
@@ -283,4 +284,28 @@ public static void info(final ObPayload payload, String database, String tableNa
                 params, result, routeTableTime, executeTime));
         }
     }
+
+    /**
+     * for tablet op
+     */
+
+    private static void logMessage0(final ObPayload payload, String database, String tableName, String methodName,
```

Comment:
Should we use `logTabletOpMessage()` instead

Reply:
done